### PR TITLE
Prometheus: Convert Prometheus query response to dataframes

### PR DESCRIPTION
**Pull request by dsotirakis**
**What this PR does / why we need it**:

The purpose of this PR is to convert query response directly to dataframes instead of time series

**Special notes for your reviewer**:

To test, set up a panel with a Prometheus datasource (need a prometheus service also running and possibly a simple node exporter).

Then, while editing the panel. add a query and an expression pointing to that query, to be able to post queries into the handler, instead of having datasources been proxied.

Then, run against master and against this branch and compare results.

### CODE REVIEW

File: pkg/tsdb/prometheus/prometheus.go [hunk 1]
```diff
@@ -193,32 +193,32 @@ func (e *PrometheusExecutor) parseQuery(dsInfo *models.DataSource, query plugins
...
-			series.Tags[string(k)] = string(v)
+			tags[string(k)] = string(v)
 		}
 
 		for _, k := range v.Values {
-			series.Points = append(series.Points, plugins.DataTimePoint{
-				null.FloatFrom(float64(k.Value)),
-				null.FloatFrom(float64(k.Timestamp.Unix() * 1000)),
-			})
+			timeVector = append(timeVector, time.Unix(k.Timestamp.Unix(), 0))
```
Comment (dsotirakis) [inline pkg/tsdb/prometheus/prometheus.go:214]: Do we need milliseconds?

Comment (wbrowne) [reply pkg/tsdb/prometheus/prometheus.go:214]: Yep, I believe so - at least according to how things were before. 

But before we were dealing in straight UNIX timestamps, now with introducing `time.Time` and therefore locale - I wonder if this is something we need be concerned about

Comment (dsotirakis) [reply pkg/tsdb/prometheus/prometheus.go:214]: @wbrowne All I found while searching for it was this one here https://github.com/grafana/grafana/blob/master/pkg/plugins/tsdb.go#L304

which basically isn't the exact milliseconds, just a convention?

**Review by wbrowne — APPROVED** [review]
Great stuff 👍

**Review by ivanahuckova — APPROVED** [review]
Great job Dimitris! I have tested this with alerting queries and for both - master and this branch - I got the same result. Good job! 🚀 LGTM.

**Pull request merged**